batch/batch/worker/worker.py

```diff
@@ -260,9 +262,9 @@ def __init__(self, job, name, spec):
             image_ref.tag = 'latest'
 
         if image_ref.name() in HAIL_GENETICS_IMAGES:
-            image_ref.domain = 'gcr.io'
-            image_ref.path = f'{PROJECT}/{image_ref.name()}'
-
+            image_ref.domain = DOCKER_PREFIX.split('/')[0]
+            image_ref.path = '/'.join(DOCKER_PREFIX.split('/')[1:] + [image_ref.path])
+                     
```

Comment:
extra white space is failing style checks

Reply:
Argh, an artefact of resolving conflicts with the GitHub UI :) Fixed.